src/main/java/org/thingsboard/trendz/generator/solution/energymetering/EnergyMeteringSolution.java

```diff
@@ -770,7 +801,9 @@ private Building makeBuildingByConfiguration(BuildingConfiguration configuration
                         .computeIfAbsent(floor, key -> new HashMap<>())
                         .computeIfAbsent(number, key -> configuration.getDefaultApartmentConfiguration());
 
-                Apartment apartment = createApartmentByConfiguration(apartmentConfiguration, configuration.getName(), floor, number, configuration.getApartmentsByFloorCount(), skipTelemetry);
+                Apartment apartment = createApartmentByConfiguration(
+                        apartmentConfiguration, configuration.getName(), floor, number, configuration.getApartmentsByFloorCount(), skipTelemetry,
+                        fullTelemetryGeneration, startGenerationTime, endGenerationTime);
```

Comment:
Almost - ")" always on the new row, add \n after "number" arg

Reply:
how many arguments do you want to be in separate row ?

Reply:
> Almost - ")" always on the new row
what ?